include/dbg_macro/dbg.h

```diff
@@ -269,7 +269,7 @@ pretty_print(std::ostream& stream, Container const& value) {
   const size_t size = value.size();
   const size_t n = std::min(size_t{5}, size);
   size_t i = 0;
-  for (auto it = begin(value); it != end(value) && i < n; ++it, ++i) {
+  for (auto it = std::begin(value); it != std::end(value) && i < n; ++it, ++i) {
```

Comment:
This breaks ADL.

Reply:
You could put `using std::begin;` and `using std::end;` in front.

Reply:
Well, I had no idea. I simply dont use the `using namespace` thing for my reasons.

Reply:
I just read [this](https://stackoverflow.com/questions/31604984/is-using-stdbegin-a-good-practice). It doesn't really convince me but I will do it anyway!

Reply:
Done